guidelines/sc/21/reflow.html

```diff
@@ -13,9 +13,9 @@ <h4>Reflow</h4>
    </ul>
    <p>Except for parts of the content which require two-dimensional layout for usage or meaning.</p>
   
-   <p class="note">320 CSS pixels is equivalent to a starting viewport width of 1280 CSS pixels wide at 400% zoom. For web content which are designed to scroll horizontally (e.g. with vertical text), the 256 CSS pixels is equivalent to a starting viewport height of 1024px at 400% zoom.</p>
+   <p class="note">320 CSS pixels is equivalent to a starting viewport width of 1280 CSS pixels wide at 400% zoom. For web content which is designed to scroll horizontally (e.g., with vertical text), 256 CSS pixels is equivalent to a starting viewport height of 1024px at 400% zoom.</p>
    
-   <p class="note">Examples of content which require two-dimensional layout are images, maps, diagrams, 
+   <p class="note">Examples of parts of content which require two-dimensional layout are images, maps, diagrams, 
```

Comment:
I don't think that this is necessary. In WCAG content can be a piece of the page (see the "change of context" def for an example).

Reply:
If you don't put in "parts of" then it needs to be "requires" > "Examples of content which requires..."

Reply:
I'd change the verb to "requires"

Reply:
I prefer the change to "requires" as well.
```suggestion
   <p class="note">Examples of content which requires two-dimensional layout are images, maps, diagrams, 
```